### ISSUE TITLE
logits_processor not exposed for SamplingConfig in high-level API

### ISSUE THREAD

**Issue by DreamGenX** [NVIDIA/TensorRT-LLM#1405]
### System Info

It would be grea to be able to access the full set of SamplingParams when using the high-level API, including SamplingConfig. This would make migration from vLLM much easier.

### Information

- [ ] The official example scripts
- [ ] My own modified scripts

### Tasks

- [ ] An officially supported task in the `examples` folder (such as GLUE/SQuAD, ...)
- [ ] My own task or dataset (give details below)

### Reproduction

N/A -- this is a feature request

### Expected behavior

N/A -- this is a feature request

### actual behavior

N/A -- this is a feature request

### additional notes

N/A -- this is a feature request

**Comment by DreamGenX** [comment]
It looks like LogitsProcessor is only available for the generate/batch API, and not for executor.

**Comment by amukkara** [comment]
@DreamGenX logits processor is now available in LLM API.

can you close this issue?

**Comment by poweiw** [comment]
Closing as the issue is stale and feature is now implemented based on https://github.com/NVIDIA/TensorRT-LLM/issues/1405#issuecomment-2477171290. Feel free to reopen @DreamGenX
